src/pages/Participate/Participate.js

```diff
@@ -1,7 +1,118 @@
 import React from 'react';
+import { useResetRecoilState } from 'recoil';
+import { useState } from 'react';
+import { setModal } from '../../recoil';
+import * as S from './Participate.sylte';
 
-const Participate = () => {
-  return <div>Participate</div>;
+const Participate = ({ data }) => {
+  const closeModal = useResetRecoilState(setModal);
+  //모달 오픈시 초기 데이터
+  const { starting_bid } = data.info;
+  const { currentPrice, setBiddingPrice } = data;
+
+  //금액을 입력할 때 설정되는 상태값들
+  const [price, setPrice] = useState('');
+  const [checkedOk, setCheckedOk] = useState(true);
+  const [bidActivation, setBidActivation] = useState(false);
```

Comment:
```suggestion
  const [checkedOk, setCheckedOk] = useState(true);
  const [bidActivation, setBidActivation] = useState(false);
```
boolean type은 보통은 의문문 형태로 짓는데, 더 자세한 내용은 이 [아티클](https://soojin.ro/blog/naming-boolean-variables)을 참고 해 보세요

Reply:
boolean type의 변수는 앞에 [is] 키워드를 추가하여 의문문형태로 변경하였습니다.